x-pack/test/plugin_functional/config.ts

```diff
@@ -43,12 +40,6 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
       serverArgs: [
         ...xpackFunctionalConfig.get('kbnTestServer.serverArgs'),
         ...plugins.map((pluginDir) => `--plugin-path=${resolve(__dirname, 'plugins', pluginDir)}`),
-        `--plugin-path=${resolve(
-          KIBANA_ROOT,
-          'test/plugin_functional/plugins/core_provider_plugin'
```

Comment:
@michaelolo24 @pgayvallet This part of the config seems to be causing trouble when running the tests locally.  I think this is because `node scripts/build_kibana_platform_plugins.js` doesn't build this plugin by default. 

To verify you can checkout `master`.
```
git checkout master
```
Then delete the build of that plugin (if you have it, I did.)
```
rm -r test/plugin_functional/plugins/core_provider_plugin/target
```
then run this command, which needs to build the plugins.
```
node scripts/build_kibana_platform_plugins
```
Then
```
yarn test:ftr --config x-pack/test/plugin_functional/config.ts
```
At that point you'll get errors related to this plugin. 
![image](https://user-images.githubusercontent.com/35559/97625688-34e96d00-19ff-11eb-9658-ed2f99f942ad.png)

![image](https://user-images.githubusercontent.com/35559/97625708-3a46b780-19ff-11eb-8d4e-64c8dfc7c8b3.png)


If you then run the build script with this flag:
```
node scripts/build_kibana_platform_plugins.js --scan-dir test/plugin_functional/
```

and then run the FTR again,
```
yarn test:ftr --config x-pack/test/plugin_functional/config.ts
```
Things work this time.

Reply:
Thank you 🙏